packages/terra-site/src/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="Content-Language" content="en-US">
```

Comment:
This is to avoid the French translation popup. In chrome, if the page contains not enough information, chrome may consider the application is in French. Other information is included in your post link.